Kickstarter-iOS/AppDelegate.swift

```diff
@@ -263,6 +263,30 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
         self?.viewModel.inputs.userSessionEnded()
       }
 
+    self.viewModel.outputs.configureSegment
+      .observeValues { writeKey in
+        let client = Analytics.configuredClient(withWriteKey: writeKey)
+        AppEnvironment.current.ksrAnalytics.configureSegmentClient(client)
+      }
+
+    self.viewModel.outputs.segmentIsEnabled
+      .observeValues { enabled in
+        if enabled {
+          Analytics.shared().enable()
+        } else {
+          Analytics.shared().disable()
+        }
+      }
+
+    NotificationCenter.default
```

Comment:
Ordinarily we would hang on to the token for these observers and release them but the `AppDelegate` is never released so I guess it's ok here 😄